release-team/role-handbooks/ci-signal.md

```diff
@@ -42,7 +42,7 @@ Here are some good early deliverables from the CI Signal lead between start of t
 - Maintain a master tracking sheet and keep it up-to-date with issues tracking any test failure/flake - [Sample sheet](https://docs.google.com/spreadsheets/d/1j2K8cxraSp8jZR2S-kJUT6GNjtXYU9hocNRiVUGZWvc/edit#gid=127492362)
 - Copy over any open test issues from previous release (ask previous CI Signal lead for the tracker sheet) and follow up on them with owners
 - Monitor [master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) dashboards **twice a week** and ensure all failures and flakes are tracked via open issues
-  - Make sure all open issues have a kind/bug, priority/flake or priority/failing-test, priority/important-soon labels 
+  - Make sure all open issues have a priority/critical-urgent label, and either a kind/bug, kind/flake or kind/failing-test label 
```

Comment:
I think for a test failure issue, we definitely need either kind/flake or kind/failing-test.  I think kind/bug is optional. wdyt?

Reply:
I'm also totally fine dropping kind/bug entirely.. I read it as "must have priority, and one of these three kinds"

Reply:
I would like it to read as "must have priority and one of either kind/flake or kind/failing-test". I am ok with dropping kind/bug as well.

/lgtm